src/style.css

```diff
@@ -1,3 +1,58 @@
-body {
+* {
   margin: 0;
 }
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 1200px;
+  /*background-color: #444;*/
+}
+
+.nav_link {
+  display: inline-block;
+  height: 56px;
+  line-height: 56px;
+  /*background-color: #444;*/
+}
+
+.logo_img {
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 250px;
+  /*background-color: #444;*/
```

Comment:
Remove redundant comments to keep your code clean

Reply:
done